ARCH-882 - Initial action for processing cypress test results (#1)
im-open/process-cypress-test-results: README.md

# process-cypress-test-results

This action works in conjunction with another step that runs `cypress run --reporter mochawesome `.  This action takes the generated results json file and creates a Status Check or PR Comment depending on the flags set with the test outcome.  

This action does not run the Cypress tests itself and it can only process one results file at a time.

* [Failures](#failures)
* [Limitations](#limitations)
* [Action Outputs](#action-outputs)
* [Inputs](#inputs)
* [Outputs](#outputs)
* [Usage Examples](#usage-examples)
* [Recompiling](#recompiling)

## Failures

> The status check can be seen as a new item on the workflow run, a PR comment or on the PR Status Check section.  If the test results contain failures, the status check will be marked as failed. Having the status check marked as failed will prevent PRs from being merged. If this status check behavior is not desired, the `ignore-test-failures` input can be set and the outcome will be marked as neutral if test failures are detected. The status badge that is shown in the comment or status check body will still indicate it was a failure though.

Reviewer: ```suggestion
The status check can be an item on the workflow run, a PR comment or on the PR Status Check section.  If the test results contain failures, the status check will be marked as failed. Having the status check marked as failed will prevent PRs from being merged. If this status check behavior is not desired, the `ignore-test-failures` input can be set and the outcome will be marked as neutral if test failures are detected. The status badge in the comment or status check body will still indicate a failure.
```
